sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/VMTestBase.cs

```diff
@@ -161,7 +164,7 @@ protected DiskEncryptionSettings GetEncryptionSettings(bool addKek = false)
 
         protected string getDefaultDiskEncryptionSetId()
         {
-            return "/subscriptions/0296790d-427c-48ca-b204-8b729bbd8670/resourceGroups/longrunning-eastus2RG/providers/Microsoft.Compute/diskEncryptionSets/longlivedBvtDES";
+            return "/subscriptions/e37510d7-33b6-4676-886f-ee75bcc01871/resourceGroups/RGforSDKtestResources/providers/Microsoft.Compute/diskEncryptionSets/EncryptionSetforTest";
```

Comment:
Not sure what this is for.  This seems to be bypassing the test recording system, and is another thing the next dev will have to change when the api-version is updated again.

Reply:
I think this is due to us not knowing how to create a DiskEncryptionSetId dynamically. Similar issue to above. This needs to be changed when tests are re-recorded. These tests will be curated and improved in the future.